I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script/script_4.rpy", line 29, in script
    s   "Dingin sekali sikap Darling padaku, apa kau lupa dengan janjimu?"
IOError: Couldn't find file 'music/bensound-love.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script/script_4.rpy", line 29, in script
    s   "Dingin sekali sikap Darling padaku, apa kau lupa dengan janjimu?"
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/ast.py", line 624, in execute
    renpy.exports.say(who, what, interact=self.interact, *args, **kwargs)
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/exports.py", line 1173, in say
    who(what, *args, **kwargs)
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/character.py", line 911, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/character.py", line 747, in do_display
    **display_args)
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/character.py", line 523, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/ui.py", line 287, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/display/core.py", line 2553, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/display/core.py", line 3104, in interact_core
    renpy.audio.audio.periodic()
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/audio/audio.py", line 960, in periodic_thread_main
    periodic_pass()
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/audio/audio.py", line 917, in periodic_pass
    c.periodic()
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/audio/audio.py", line 409, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/audio/audio.py", line 69, in load
    rv = renpy.loader.load(fn)
  File "/home/hernowo/Data_Hernowo/Program/Linux/renpy-6.99.13-sdk/renpy/loader.py", line 548, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'music/bensound-love.mp3'.

Linux-4.13.0-41-generic-x86_64-with-debian-stretch-sid
Ren'Py 6.99.13.2919
Cengoh Software 1.0
